packages/alert-banner/index.test.js

```diff
@@ -44,20 +52,22 @@ describe('<AlertBanner />', () => {
       Date.now.mockRestore()
     })
 
-    it('should show the banner when the current date has not surpassed the expiration date', () => {
+    it('should show the banner before the expiration date', () => {
       const expirationDate = '2020-10-30T12:00:00-07:00'
       const { container } = render(
         <AlertBanner text="text" tag="tag" expirationDate={expirationDate} />
       )
-      expect(container.firstChild).toHaveClass('show')
+      const rootElem = container.firstChild
+      expect(rootElem.getAttribute('data-test-is-shown')).toBe('true')
     })
 
-    it('should NOT show the banner when the current date has surpassed the expiration date', () => {
+    it('should hide the banner after the expiration date', () => {
       const expirationDate = '2020-10-01T12:00:00-07:00'
       const { container } = render(
         <AlertBanner text="text" tag="tag" expirationDate={expirationDate} />
       )
-      expect(container.firstChild).not.toHaveClass('show')
+      const rootElem = container.firstChild
+      expect(rootElem.getAttribute('data-test-is-shown')).toBe('false')
```

Comment:
We should still be able to assert on a class being present, we use https://www.npmjs.com/package/identity-obj-proxy in tests, so whatever key we're reading will be returned.

```jsx
<div className={s.isShown} /> // -> <div class="isShown" />
```

Reply:
Oh nice, that's much better 💯 Will update 👍

Reply:
✅ Updated